src/data/LedgerCache.hpp

```diff
@@ -86,7 +86,7 @@ class LedgerCache {
     /**
      * @brief Update the cache with new ledger objects.
      *
-     * @param blobs The ledger objects to update cache with
+     * @param objs The ledger objects to update cache with
```

Comment:
Will the inconsistency between documentation and parameter name result to a warning or error in doxygen?
Is it possible to automatically detect such inconsistencies?

Reply:
Only in warnings. We could make the into errors if we wanted to. We have disabled some of that stuff in Doxyfile atm.

Reply:
Shall we change it to have errors?